### PR TITLE
Alpine 3.13

### PR DESCRIPTION
**Pull request by J0WI**

### CODE REVIEW

File: 2.4-rc/Dockerfile [hunk 1]
```diff
@@ -12,9 +12,9 @@ RUN set -eux; \
 		--uid 99 \
 		haproxy
 
-ENV HAPROXY_VERSION 2.4-dev5
-ENV HAPROXY_URL https://www.haproxy.org/download/2.4/src/devel/haproxy-2.4-dev5.tar.gz
-ENV HAPROXY_SHA256 fcb2f70c69bbf40b6acff3a8745b89ac4c5d29f6b015d16e605eccea7cee00ab
+ENV HAPROXY_VERSION 2.4-dev6
+ENV HAPROXY_URL https://www.haproxy.org/download/2.4/src/devel/haproxy-2.4-dev6.tar.gz
+ENV HAPROXY_SHA256 1ae9b59128725ee538734fc51296b3b4e8582bd616bb151e33824038111ec4f4
```
Comment (tianon) [inline 2.4-rc/Dockerfile:17]: :eyes: :question:

File: update.sh [hunk 2]
```diff
@@ -13,9 +13,9 @@ defaultDebianSuite='buster-slim'
 declare -A debianSuite=(
 	#[1.6]='stretch-slim'
 )
-defaultAlpineVersion='3.12'
+defaultAlpineVersion='3.13'
 declare -A alpineVersion=(
-	#[1.6]='3.8'
+	[1.7]='3.12'
```
Comment (tianon) [inline update.sh:18]: This is interesting - was there a build failure on HAProxy 1.7?
(maybe a one-liner error we could include in a comment here?)

File: 2.4-rc/Dockerfile (hunk 1 above)
Comment (J0WI) [reply 2.4-rc/Dockerfile:17]: superseded by 63151039103973f61addbfd90c7eeb027d916682

File: update.sh (hunk 2 above)
Comment (J0WI) [reply update.sh:18]:
```
multiple definition of `pool2_trash'; src/haproxy.o:(.bss+0x0): first defined here
collect2: error: ld returned 1 exit status
```
Alpine 3.13 upgraded to GCC 10

**Review by yosifkit — APPROVED** [review]

**Pull request merged**